src/main/java/org/openelisglobal/testconfiguration/controller/TestModifyEntryController.java

```diff
@@ -219,6 +219,8 @@ public int compare(ResultLimit o1, ResultLimit o2) {
                     significantDigits, "-"));
             bean.setReportingRange(SpringContext.getBean(ResultLimitService.class).getDisplayReportingRange(limit,
                     significantDigits, "-"));        
+            bean.setReportingRange(SpringContext.getBean(ResultLimitService.class).getDisplayCriticalRange(limit,
```

Comment:
I assume we don’t want to overwrite the reportingRange with the critical range, but instead mean to mirror the finctionality here by adding a g/setHighCriticalRange and g/setLowCriticalRange to ResultLimitBean?